packages/mustache/mustache.3.2.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: "Mustache logic-less templates in OCaml"
+description: """
+
+Read and write mustache templates, and render them by providing a json object.
+Contains the `mustache` command line utility for driving logic-less templates.
+"""
+maintainer: ["Rudi Grinberg <me@rgrinerg.com>"]
+authors: [
+  "Rudi Grinberg <me@rgrinberg.com>"
+  "Armaël Guéneau <armael.gueneau@ens-lyon.fr>"
+  "Gabriel Scherer <gabriel.scherer@gmail.com>"
+]
+license: "MIT"
+homepage: "https://github.com/rgrinberg/ocaml-mustache"
```

Comment:
(triage note) it seems that the doc link (`http://rgrinberg.github.io/mustache/`) is removed. it's likely intentional since the link is now 404

Reply:
doc generation for the current version of the packageis broken. It should be fixed for next version. I'll change the url to one pointing to the official ocaml docs (even if they are empty for the time being)